data/abilities.ts

```diff
@@ -184,7 +184,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			this.add('-ability', pokemon, 'Unnerve');
 			this.effectState.unnerved = true;
 		},
-		onEnd() {
+		onEnd(pokemon) {
```

Comment:
Why add a variable you don't use?

Reply:
I have a few things in this pr that i still have to clean up, been super busy with exams/work/college stuff so i put it off. hoping to finish it this weekend